quantum/bits.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+#pragma once
+
+#define BITS_PER_BYTE 8
+#define BITS_PER_LONG 32
+#define BITS_PER_LONG_LONG 64
+
+#define BIT(nr) (UL(1) << (nr))
+#define BIT_ULL(nr) (ULL(1) << (nr))
+#define BIT_MASK(nr) (UL(1) << ((nr) % BITS_PER_LONG))
+#define BIT_WORD(nr) ((nr) / BITS_PER_LONG)
+#define BIT_ULL_MASK(nr) (ULL(1) << ((nr) % BITS_PER_LONG_LONG))
+#define BIT_ULL_WORD(nr) ((nr) / BITS_PER_LONG_LONG)
+
+/*
+ * Create a contiguous bitmask starting at bit position @l and ending at
+ * position @h. For example
```

Comment:
Document that the `h` and `l` endpoints are inclusive (if I got that right). Also, same comment as above that suffixing with the type's width "32" or "64" would be clearer.

Reply:
Done.